remove how-to from README.md (#66)
Percona-Lab/percona-server-mongodb-operator: README.md

[![Build Status](https://travis-ci.org/Percona-Lab/percona-server-mongodb-operator.svg?branch=master)](https://travis-ci.org/Percona-Lab/percona-server-mongodb-operator)
[![Go Report Card](https://goreportcard.com/badge/github.com/Percona-Lab/percona-server-mongodb-operator)](https://goreportcard.com/report/github.com/Percona-Lab/percona-server-mongodb-operator)
[![codecov](https://codecov.io/gh/Percona-Lab/percona-server-mongodb-operator/branch/master/graph/badge.svg)](https://codecov.io/gh/Percona-Lab/percona-server-mongodb-operator)

A Kubernetes operator for [Percona Server for MongoDB](https://www.percona.com/software/mongo-database/percona-server-for-mongodb) based on the [Operator SDK](https://github.com/operator-framework/operator-sdk).

# Documentation
See the [Official Documentation](https://percona-lab.github.io/percona-server-mongodb-operator/) for more information.

[![Official Documentation](https://via.placeholder.com/260x60/419bdc/FFFFFF/?text=Documentation)](https://percona-lab.github.io/percona-server-mongodb-operator/)

# DISCLAIMER

**This code is incomplete, expect major issues and changes until this repo has stabilised!**

## Submitting Bug Reports

> If you find a bug in Percona Docker Images or one of the related projects, you should submit a report to that project's [JIRA](https://jira.percona.com) issue tracker.

Reviewer: ```suggestion
If you find a bug in Percona Docker Images or in one of the related projects, please submit a report to that project's [JIRA](https://jira.percona.com) issue tracker.
```

> Your first step should be [to search](https://jira.percona.com/issues/?jql=project%20%3D%20%22Cloud%20Dev%22) the existing set of open tickets for a similar report. If you find that someone else has already reported your problem, then you can upvote that report to increase its visibility.

Reviewer: ```suggestion
Your first step should be [search](https://jira.percona.com/issues/?jql=project%20%3D%20%22Cloud%20Dev%22)  for a similar report in the existing set of open tickets. If someone else has already reported your problem, upvote that report to increase its visibility.
```
